CleverPush/Source/CPAppBannerModuleInstance.m

```diff
@@ -553,27 +554,53 @@ - (void)scheduleBanners {
         return;
     }
 
-    for (CPAppBanner* banner in [self getActiveBanners]) {
-        if (!banner.startAt || [banner.startAt compare:[NSDate date]] == NSOrderedAscending) {
-            if (banner.delaySeconds > 0) {
-                dispatch_after(dispatch_time(DISPATCH_TIME_NOW, NSEC_PER_SEC * banner.delaySeconds), dispatch_get_global_queue(DISPATCH_QUEUE_PRIORITY_DEFAULT, 0), ^(void) {
-                    [self showBanner:banner force:NO];
-                });
+    if (currentEventId != nil && ![currentEventId isKindOfClass:[NSNull class]] && ![currentEventId isEqualToString:@""] ) {
```

Comment:
Please adapt everything to latest changes from here: https://github.com/cleverpush/cleverpush-android-sdk/pull/164/commits/00783e977e8ab2cb362a70a327f97b6e0dd1466a
And connect with Unnati to validate.